share/test/interop/bin/test_rpc_interop.sh

```diff
@@ -21,29 +21,33 @@ cd `dirname "$0"`/../../../..   # connect to root
 
 VERSION=`cat share/VERSION.txt`
```

Comment:
```bash
VERSION=$(<share/VERSION.txt)
```

would be a little nicer here. (Obviously take it or leave it since this isn't actually part of this PR.)